src/ducks/Signals/utils/utils.js

```diff
@@ -34,7 +34,7 @@ import {
   TIME_WINDOW_UNITS,
   getDefaultTimeRangeValue
 } from './constants'
-import { capitalizeStr } from '../../../utils/utils'
+import { capitalizeStr, isEthStrictAddress } from '../../../utils/utils'
```

Comment:
Is this doing the checksum (upper and lower case matter), too? Will it accept all lowercase or all uppercase because the backend accepts such addresses